java/java.lexer/nbproject/project.properties

```diff
@@ -21,6 +21,11 @@ javac.source=1.8
 javadoc.title=Java Lexer API
 javadoc.apichanges=${basedir}/apichanges.xml
 
+test.config.default.excludes=\
+    **/JavaLexerPerformanceTest.class,\
+    **/JavaFlyTokensTest.class,\
+    **/JavaTokenDumpTest.class
```

Comment:
those `java.lexer` tests require a test file (looks like it was the source code of JComponent) which probably has been removed before the code was donated.

Reply:
The `JavaTokenDumpTest` can be fixed like this:

```diff
--- a/java/java.lexer/test/unit/data/testfiles/testInput.java.txt.tokens.txt
+++ b/java/java.lexer/test/unit/data/testfiles/testInput.java.txt.tokens.txt
@@ -41,7 +41,7 @@
 ----- EOF -----
 
 .t.e.s.t. String Literals
-STRING_LITERAL  """"
+STRING_LITERAL  """", la=1
 WHITESPACE      " ", la=1
 STRING_LITERAL  ""a""
 WHITESPACE      "\n", la=1
```

It also looks sane. When the lexer works on `"|" "a"` (| marks read position) and reads the next char `"`, it has to decide whether this is the end of an old style string (single quote) or the second quote char of a multiline string start `"""`. So with the addition of multiline strings look-ahead became necessary.

Reply:
awesome. done.